packages/yoastseo/src/scoring/assessments/inclusiveLanguage/configuration/cultureAssessments.js

```diff
@@ -44,9 +44,14 @@ const cultureAssessments = [
 	{
 		identifier: "exotic",
 		nonInclusivePhrases: [ "exotic" ],
-		inclusiveAlternatives: "<i>unfamiliar, foreign, peculiar, fascinating, alluring, bizarre</i>",
-		score: SCORES.NON_INCLUSIVE,
-		feedbackFormat: potentiallyHarmful,
+		inclusiveAlternatives: "<i>unfamiliar, foreign, peculiar, fascinating, alluring, bizarre, nonnative, introduced</i>",
```

Comment:
I wonder which spelling is actually standard for "nonnative"? I checked [Merriam Webster dictionary for "nonnative" entry](https://www.merriam-webster.com/dictionary/nonnative) and it redirects me to "non-native".

Reply:
Yes, nonnative is actually a lazy and not grammatically correct spelling that seems like i appropriated from reading about introduced species and plants🙈 Will fix it :)